### DISCUSSION TITLE
Upcoming change to context of `view/title` menu

### DISCUSSION THREAD

**Discussion by isidorn** [microsoft/vscode-discussions#121] [Announcements]
In August we tried passing the focus and selection of the view's tree for commands contributed to the `view/title` menu of a tree view. For this release we have reverted the change as it caused problems for some extensions. We want to bring the change back for September when we plan to release it. If this will break an extension or if this is otherwise undesirable behavior, please let us know by commenting here

fyi @alexr00 @benibenj

**Comment by benibenj** [comment]
As discussed in [this](https://github.com/microsoft/vscode/issues/42903) feature request, we will provide the focused and selected tree items as arguments when running `view/title` commands similar to how we do this with context menu commands. 

We had planned to release this feature in August, however, the change ended up breaking commands of some extensions as they reused the same commands in many areas and made assumptions about the arguments they would receive. With our change we broke their assumptions. 
We decided to revert the change for August and will bring it back in the September release, giving extension authors more time to adapt to the new `view/title` command argument. The change should be in insiders by next week.
